src/js/controller/controller.js

```diff
@@ -511,6 +511,10 @@ Object.assign(Controller.prototype, {
                     });
                 }
 
+                if (_this.getMute() && _model.get('enableCaptionsOnMutedAutostart')) {
```

Comment:
Why not just clear "enableCaptionsOnMutedAutostart"? We don't need another property.